apps/condo/domains/ticket/schema/TicketAnalyticsReportService.test.js

```diff
@@ -2,22 +2,19 @@
  * Generated by `createservice ticket.TicketAnalyticsReportService`
  */
 
-const { TICKET_ANALYTICS_REPORT_MUTATION } = require(('@condo/domains/ticket/gql'))
+const { TICKET_ANALYTICS_REPORT_QUERY } = require(('@condo/domains/ticket/gql'))
 const moment = require('moment')
 const { makeClientWithProperty } = require('@condo/domains/property/utils/testSchema')
 const { createTestTicket, makeClientWithTicket } = require('@condo/domains/ticket/utils/testSchema')
-const { TICKET_STATUS_TYPES } = require('@condo/domains/ticket/constants')
 const { makeClient } = require('@core/keystone/test.utils')
-const DATE_FORMAT = 'DD.MM.YYYY'
-const NOW_DATE = moment().format(DATE_FORMAT)
 
 describe('TicketAnalyticsReportService', () => {
     describe('User', () => {
         it('can read TicketAnalyticsReportService grouped counts [day, status]', async () => {
```

Comment:
Could you please write a test case with more than one ticket and inspect count of grouped result items.

Reply:
@AntonAL Done